pkg/server/test/check.go

```diff
@@ -146,15 +391,105 @@ func BenchmarkCheck(b *testing.B, ds storage.OpenFGADatastore) {
 		b.Run(name, func(b *testing.B) {
 			for i := 0; i < b.N; i++ {
 				response, _, err := checkQuery.Execute(ctx, &openfgav1.CheckRequest{
-					StoreId:          storeID,
-					TupleKey:         bm.tupleKey,
-					ContextualTuples: bm.contextualTuples,
-					Context:          bm.contextStruct,
+					StoreId:  storeID,
+					TupleKey: bm.tupleKeyToCheck,
+					Context:  bm.contextGenerator(),
 				})
 
 				require.Equal(b, bm.expected, response.GetAllowed())
 				require.NoError(b, err)
```

Comment:
these two lines should be in the opposite order

Reply:
indeed they should